cohttp-curl-lwt/bin/dune

```
@@ -0,0 +1,12 @@
+(executable
+ (name curl)
+ (libraries
+  sexplib0
+  cohttp
+  cohttp_curl_lwt
+  lwt.unix
+  lwt
+  logs
+  logs.cli
+  fmt.tty
+  cmdliner))
```

```suggestion
  cmdliner)
  (public_name cohttp-curl)
  (package cohttp-curl-lwt))
```
Should we install the binary like cohttp-lwt-unix does? The name omits lwt since it may conflict with cohttp-lwt-unix i stalled binaries. We should probably reconsider installing them at all or create a separate `bin` package. 

It could be part of a separate pr, not asking to do it here, but I want to understand what to do with this binary: does a cohttp-async install with test try to build this package as well for example or dune is smart enough to ignore it?

---

> Should we install the binary like cohttp-lwt-unix does? The name omits lwt since it may conflict with cohttp-lwt-unix i stalled binaries. We should probably reconsider installing them at all or create a separate bin package.

Probably not. I don't see  a use for it other than testing.

> It could be part of a separate pr, not asking to do it here, but I want to understand what to do with this binary: does a cohttp-async install with test try to build this package as well for example or dune is smart enough to ignore it?

Dune is smart enough to ignore it. Since the binary doesn't belong to any package, it will not be mentioned in the .install file in any way and dune will not build it.